src/app/router.tsx

```diff
@@ -16,6 +19,14 @@ const router = createHashRouter([
     path: '/auth/login',
     element: <Login />,
   },
+  {
+    path: '/account',
+    element: (
+      <PrivateRoute isLoggedIn={isLoggedIn}>
```

Comment:
since isLoggedIn logic is always going to be dependent on if user object exists in state.auth, do we need extra props here? no need open for extension here

Reply:
开关 测试效果 可改